### PR TITLE
Repo maintenance

### PR DESCRIPTION
**Pull request by wow-such-code**
I assume code QL will work once this is on dev?

### CODE REVIEW

File: .github/workflows/changelog-update.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  # Enforces the update of a changelog file on every pull request
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dangoslen/changelog-enforcer@v2
+        with:
+          changeLogPath: 'CHANGELOG.rst'
+          skipLabels: 'Skip-Changelog'
```
Comment (jenniferboedker) [inline .github/workflows/changelog-update.yml]: A file should always end with an empty new line
```suggestion
          skipLabels: 'Skip-Changelog'
          
```

File: .github/workflows/nexus-publish-release.yml [hunk 2]
```diff
@@ -30,11 +29,14 @@ jobs:
...
+
     -   name: Build with Maven
         run: mvn -B package --file pom.xml
 
     -   name: Publish artefact to QBiC Nexus Repository
         run: mvn --quiet --activate-profiles !development-build,release-build --settings $GITHUB_WORKSPACE/.github.settings.xml deploy
         env:
             MAVEN_REPO_USERNAME: ${{ secrets.NEXUS_USERNAME }}
-            MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+            MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
```
Comment (jenniferboedker) [inline .github/workflows/nexus-publish-release.yml]:
```suggestion
            MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
            
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,21 @@ Changelog
 This document records all notable changes to `HTTPie <https://httpie.org>`_.
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.7.0 (2022-02-07)
+------------------
+
+**Added**
+
+* Update github workflows
+* Update License
```
Comment (jenniferboedker) [inline CHANGELOG.rst]: I think you dont need to mention this
```suggestion
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,21 @@ Changelog
...
+* Remove reference to parent pom
```
Comment (jenniferboedker) [inline CHANGELOG.rst]: You did not remove the parent pom 😅  At least I cant see it in the pom.. and then you also dont need to increase the version

File: .github/workflows/nexus-publish.yml [hunk 5]
```diff
@@ -16,10 +16,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 1.11
+      - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
-          java-version: 1.11
+          java-version: 1.8
```
Comment (jenniferboedker) [inline .github/workflows/nexus-publish.yml]: +1

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
You need to add this
```
     # Remove existing snapshot tags which are not supposed to be present
      - name: Remove snapshot tags
        run: mvn versions:set -DremoveSnapshot
      # Set the SNAPSHOT for this build and deployment
      - name: Set version in Maven project
        run:  mvn versions:set -DnewVersion='${project.version}-SNAPSHOT'
```
to nexus-publish.yml instead of nexus-publish-release.yml. The later one should also be deleted.

Furthermore, the release workflow is missing and you need to add caching to the build_package.yml (See https://github.com/qbicsoftware/data-model-lib/pull/286/files)

I am also not sure if you really need the version increase here, it seems to me that the parent-pom was not removed.

File: CHANGELOG.rst (hunk 4 above)
Comment (wow-such-code) [reply CHANGELOG.rst]: just removed a comment that referenced the parent pom, but you are correct about the version!

**Comment by jenniferboedker** [comment]
Since you are not changing the actual code for this repo I think this should be merged to master, also: there is still the comment about the pom and the new version in the CL. Why did you add it here?

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
Thanks for addressing my comments!
I have some more (sorry!): the workflow structure does not look like in the template (not all ymls are there, e.g also check outside the workflow folder of the template). I am happy to have a quick call to clarify whats missing.

**Comment by jenniferboedker** [comment]
I jsut also found the "how to run" and "how to use" sections are missing. I think you just need to update the section titles since the content is already there

File: .github/release.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```
Comment (sven1103) [inline .github/release.yml:6]:
```suggestion
      - JohnnyQ5
      - github-actions
```

File: .github/workflows/changelog-update.yml (hunk 1 above)
Comment (jenniferboedker) [inline .github/workflows/changelog-update.yml]: This file can be removed

File: README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+[![Build Maven Package](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/build_package.yml)
+[![Run Maven Tests](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/run_tests.yml)
+[![CodeQL](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/codeql-analysis.yml)
+[![release](https://img.shields.io/github/v/release/qbicsoftware/xml-manager-lib?include_prereleases)](https://github.com/qbicsoftware/xml-manager-lib/releases)
+
+[![license](https://img.shields.io/github/license/qbicsoftware/xml-manager-lib)](https://github.com/qbicsoftware/xml-manager-lib/blob/main/LICENSE)
+![language](https://img.shields.io/badge/language-java-blue.svg)
+
+</div>
```
Comment (jenniferboedker) [inline README.md]:
```suggestion
<div align="center">
# XML Manager Library
<i>Library for reading and writing of specific openBIS xml properties </i>

[![Build Maven Package](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/build_package.yml)
[![Run Maven Tests](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/run_tests.yml)
[![CodeQL](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/xml-manager-lib/actions/workflows/codeql-analysis.yml)
[![release](https://img.shields.io/github/v/release/qbicsoftware/xml-manager-lib?include_prereleases)](https://github.com/qbicsoftware/xml-manager-lib/releases)

[![license](https://img.shields.io/github/license/qbicsoftware/xml-manager-lib)](https://github.com/qbicsoftware/xml-manager-lib/blob/main/LICENSE)
![language](https://img.shields.io/badge/language-java-blue.svg)

</div>
```

**Review by jenniferboedker — COMMENTED** [review]
some minor things left then you are good to go :)

**Review by jenniferboedker — APPROVED** [review]
LGTM

**Pull request merged**